x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx

```diff
@@ -678,7 +678,7 @@ export class VectorLayer extends AbstractLayer implements IVectorLayer {
         syncContext,
         source,
       });
-      await this._syncIsEditable({ syncContext });
+      await this._syncSupportsFeatureEditing({ syncContext, source });
```

Comment:
What's the reason for adding `source` into this call?

Reply:
I see. Using it down below instead of calling `getSource`

Reply:
Ya, I say this when changing the names. The reason is described in this comment

https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx#L655